Header header
uint64 time_usec
uint8 system_id
float64 roll
float64 pitch
float64 yaw